.github/workflows/release.yml

```
@@ -309,12 +309,30 @@ jobs:
           cat release_notes.md
           echo "📋 End of extracted content"
           
+          # Get previous version for Full Changelog link
+          PREV_VERSION=$(git tag --sort=-version:refname | grep -v "$VERSION" | head -n1 || echo "")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)